src/modules/map/components/cesium/CesiumMap.vue

```diff
@@ -103,6 +115,10 @@ export default {
         )
         baseLayer.show = true
 
+        const rectangle = Rectangle.fromDegrees(5.1402988, 45.3981222, 11.4774363, 48.230617)
```

Comment:
Todo, use `WGS84_EXTENT` once #423 is merged

Reply:
little heads up : it is now renamed `TILEGRID_EXTENT_EPSG_4326`